services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py

```diff
@@ -93,7 +94,7 @@ def _shared_store(self) -> SharedStore:
         return self._app.state.shared_store
 
     @property
-    def compose_spec(self) -> str | None:
+    def compose_spec(self) -> Optional[str]:
```

Comment:
this also looks strange, right?

Reply:
yes it was caused by reverting changes, I have fix it in last commit now.